librz/core/cmd_analysis.c

```diff
@@ -5196,6 +5197,7 @@ static void cmd_analysis_esil(RzCore *core, const char *input) {
 		}
 		break;
 	case 'b': // "aeb"
+		// BUG : aeb causes SEGMENT FAULT
```

Comment:
In original ESIL as well? Please open an issue.

Reply:
Linked issue #1365